### PR TITLE
OP-22190: Upgrade Spinnaker 1.33.2

### PR DESCRIPTION
**Pull request by sudhakaropsmx**
https://devopsmx.atlassian.net/browse/OP-22190

Verified the basic functionality create application, create pipeline , trigger pipeline, login SAML and LDAP 
image: quay.io/opsmxpublic/ubi8-gate-cve:1.33.2-OP-22085-0307241300
Env:  https://deck-cvedev.devisd.opsmx.net/#/applications

### CODE REVIEW

File: gate-iap/src/main/java/com/netflix/spinnaker/gate/security/iap/IapSsoConfig.java [hunk 1]
```diff
@@ -102,7 +101,7 @@ public void configure(HttpSecurity http) throws Exception {
     http.addFilterBefore(iapAuthenticationFilter(), BasicAuthenticationFilter.class);
   }
 
-  @Override
+  //  @Override
```
Comment (aman-agrawal) [inline gate-iap/src/main/java/com/netflix/spinnaker/gate/security/iap/IapSsoConfig.java:104]: I dont see usage of it, can you pls verify relevance of these methods.
May be as a separate task (not a blocker for upgrade-1.33.x task)

**Comment by aman-agrawal** [comment]
why below files got deleted :
gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/RetrofitConfig.groovy
gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OidcConfigController.java
gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/NoopOidcConfigService.java
gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/OidcConfigService.java

**Comment by aman-agrawal** [comment]
Pls resolve the conflicts.
Pls add testing details & pls mention what left on UT side.

**Comment by aman-agrawal** [comment]
Verified few commits from sheet - LGTM

**Review by aman-agrawal — CHANGES_REQUESTED** [review]

File: gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/KeelService.java [hunk 2]
```diff
@@ -17,15 +17,7 @@
...
-import com.netflix.spinnaker.gate.model.manageddelivery.ConstraintState;
-import com.netflix.spinnaker.gate.model.manageddelivery.ConstraintStatus;
-import com.netflix.spinnaker.gate.model.manageddelivery.DeliveryConfig;
-import com.netflix.spinnaker.gate.model.manageddelivery.EnvironmentArtifactPin;
-import com.netflix.spinnaker.gate.model.manageddelivery.EnvironmentArtifactVeto;
-import com.netflix.spinnaker.gate.model.manageddelivery.GraphQLRequest;
-import com.netflix.spinnaker.gate.model.manageddelivery.OverrideVerificationRequest;
-import com.netflix.spinnaker.gate.model.manageddelivery.Resource;
-import com.netflix.spinnaker.gate.model.manageddelivery.RetryVerificationRequest;
+import com.netflix.spinnaker.gate.model.manageddelivery.*;
```
[the remark this answers is not in the document]
Comment (sudhakaropsmx) [reply gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/KeelService.java]: Removed * import and replaced with specific classes

File: .github/workflows/gate-build-pr-base.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+name: Build Base Branch Code on PR Raised
+
+
+on:
+  pull_request:
+    branches:
+      - OES-1.30.x
```
[the remark this answers is not in the document]
Comment (sudhakaropsmx) [reply .github/workflows/gate-build-pr-base.yml]: Updated the version gate branch

**Comment by sudhakaropsmx** [comment]
> why below files got deleted : gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/RetrofitConfig.groovy gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OidcConfigController.java gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/NoopOidcConfigService.java gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/OidcConfigService.java

These are removed from the 1.33.2

File: gate-iap/src/main/java/com/netflix/spinnaker/gate/security/iap/IapSsoConfig.java (hunk 1 above)
Comment (sudhakaropsmx) [reply gate-iap/src/main/java/com/netflix/spinnaker/gate/security/iap/IapSsoConfig.java:104]: yes, these are not blockers these same as in 1.30.x

**Comment by sudhakaropsmx** [comment]
> Pls resolve the conflicts. Pls add testing details & pls mention what left on UT side.

gate-web test case yet to verify other than this out of 70 1 failing

**Comment by sudhakaropsmx** [comment]
> Pls resolve the conflicts. Pls add testing details & pls mention what left on UT side.
91 test cases 83 passed 8 failed

**Review by j-sandy — APPROVED** [review]
LGTM !

**Comment by sudhakaropsmx** [comment]
> > Pls resolve the conflicts. Pls add testing details & pls mention what left on UT side.

Screenshot from 2024-07-03 13-51-38

**Pull request merged**